src/labels/CustomLabels.labels

```diff
@@ -134,7 +134,15 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>addrNotEnabled</shortDescription>
-        <value>Address management not enabled for this object type in settings.</value>
+        <value>The ability to add multiple addresses is disabled for this Account record type. To enable it, go to HEDA Settings, click on Accounts and Contacts, and select the appropriate checkbox next to Account Types with Multi-Addresses Enabled.</value>
+    </labels>
+    <labels>
+        <fullName>addrNotEnabledContact</fullName>
+        <categories>Address Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
```

Comment:
They should be protected unless there's a good reason for them not to be (if we wanted users to be able to change them). The reason why it's better for us to make them protected is that protected labels can be deleted from the package if we don't need them any more.